app/test_main.py

```diff
@@ -1,3 +1,44 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age, dog_age, expected", [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (24, 24, [2, 2]),
+    (28, 28, [3, 2])
+])
+def test_get_human_age(cat_age: int, dog_age: int, expected: list) -> None:
+    assert (
+        get_human_age(cat_age, dog_age) == expected
+    ), f"Test failed for cat_age={cat_age}, dog_age={dog_age}"
+
+
+@pytest.mark.parametrize("cat_age, dog_age, expected", [
```

Comment:
check your styling - it would be more readable if you start params from new line:
```
@pytest.mark.parametrize(
    "cat_age,dog_age,expected",
    [
       (-1, 10, [0, 0]),...